doc/api/crypto.md

```diff
@@ -1969,6 +1969,7 @@ added: v6.6.0
 - `a` {Buffer | TypedArray | DataView}
 - `b` {Buffer | TypedArray | DataView}
 
+This function is based on constant time algorithm.
```

Comment:
Nit: `based on a constant-time algorithm`, or `based on constant-time comparisons` would be more grammatically correct..

Reply:
@joyeecheung : agree.. let me make the correction to `based on a constant-time algorithm`

Reply:
@joyeecheung : I've made the change. Please review. Thanks.